Dockerfile.fedora-25-master-nightly

```diff
@@ -0,0 +1,87 @@
+# Clone from the Fedora 25 image
+FROM fedora:25
+
+MAINTAINER IPA Maintainers <ipa-maint@redhat.com>
+
+RUN mkdir -p /run/lock \
+    && cd /etc/yum.repos.d \
+    && curl -O https://copr.fedorainfracloud.org/coprs/g/freeipa/freeipa-master/repo/fedora-25/group_freeipa-freeipa-master-fedora-25.repo \
+    && curl -O https://copr.fedorainfracloud.org/coprs/g/freeipa/freeipa-master-nightly/repo/fedora-25/group_freeipa-freeipa-master-nightly-fedora-25.repo \
+    && dnf install -y freeipa-server \
+    freeipa-server-dns \
+    freeipa-server-trust-ad \
+    initscripts \
+    --best --allowerasing \
```

Comment:
I would rather keep it there to take care of incidental dependency conflicts that happen sometimes when using nightly builds.